branca/utilities.py

```diff
@@ -44,11 +44,12 @@ def legend_scaler(legend_values, max_labels=10.0):
     if len(legend_values) < max_labels:
         legend_ticks = legend_values
     else:
-        spacer = int(math.ceil(len(legend_values)/max_labels))
+        spacer = int(math.ceil(len(legend_values) / (max_labels - 1)))
         legend_ticks = []
         for i in legend_values[::spacer]:
             legend_ticks += [i]
             legend_ticks += ['']*(spacer-1)
+        legend_ticks += [legend_values[-1]]
```

Comment:
I understand the desire to always show the last value. But this means we may have differing distances between labels. For example consider values `[0, 1, 2, 3]` and `max_labels=3`. Should the ticks then be `[0, 2]` (current situation, equal distance between values but missing last value) or `[0, 2, 3]` (your PR, unequal distance between values but including last value)? I don't know which is better at the moment, but in doubt I tend to stick with the status quo. What do you think?

Is there a way to have both? Include the first and last value, and have equal spacing?